airbyte-integrations/connectors/source-delighted/source_delighted/spec.json

```
@@ -9,12 +9,14 @@
     "properties": {
       "since": {
         "type": "integer",
-        "description": "An Unix timestamp to retrieve records created on or after this time.",
-        "examples": [1625328167]
+	"title": "Since",
+        "description": "An optional Unix timestamp to restrict responses to those created on or after this time.",
+        "example (for 1 hour ago)": [1638540972]
```

```suggestion
        "examples": [1625328167]
```
It is necessary to leave it that way, because the `examples` field is system-wide and is displayed only if it is specified as `examples` and not as `example (for 1 hour ago)`.